Add "What makes PostHog different" FAQ to comparison blog posts (#18864)
PostHog/posthog.com: contents/blog/best-activecampaign-alternatives.mdx

Reviewer: Do you think that maybe this is too many "selling PostHog" sections in these?

Like we have:
- Why do companies use PostHog
- Bottom line
- Is PostHog right for you?
- The bottom boiler plate explanation

Then this, is it too much?

Reviewer: Like I know it will be harder, but would it be better to blend self-driving into the content that's already there? I'm thinking we could have a point in "Is PostHog right for you?"

Author: Yeah it is a bit pushy -- honestly my thinking here was that it would be nice having _something_ in place (especially in the older SEO articles which are lower in priority for optimizations and don't mention MCP/desktop/self driving stuff at all), just as a bandaid solution while we don't do updates in the body copy for all of them.

That said I agree it might be a little too salesy/self pats on the back, what about a more educational one like "What does "self-driving product" mean? The reason why I didn't take this route was that if we dont mention self driving anywhere and all of a sudden the first FAQ answer talks about it that feels awk too

## Frequently asked questions

<details>
<summary>What makes PostHog different from other tools?</summary>

PostHog makes your product [self-driving](/self-driving), and no, this isn't about cars. It digs through your product data, finds what's worth fixing, and has agents do the work. 

It starts with context. Having a ton of developer tools in one place – [product analytics](/product-analytics), [web analytics](/web-analytics), [session replay](/session-replay), [feature flags](/feature-flags), [experiments](/experiments), [error tracking](/error-tracking), [logs](/logs), [AI observability](/ai-observability), and a [data warehouse](/data-stack) – means PostHog sees everything happening in your product.

Then [scouts](/blog/what-is-a-scout) read across all of it and decide what's worth knowing and what’s just noise. What clears the bar becomes a report in your inbox, an agent picks it up, roots out the cause, and opens the PR. You review and merge.

You can steer it from [Slack](/slack), the [web app](/ai), the [desktop app](/desktop), or your own editor via [the MCP](/mcp) or [CLI](/docs/cli). Dealer's choice.

> 

Author: ```suggestion
```

## Frequently asked questions

<details>
<summary>What makes PostHog different from other tools?</summary>

> PostHog helps you build [self-driving products](/self-driving), and no, this isn't about cars. Instead of you prompting an agent, the product prompts itself.

Reviewer: PostHog makes your product self-driving. Also, the product doesn't prompt itself. It's more like the product improves itself without you asking. 

Copying the copy from here: https://posthog.com/self-driving

```suggestion
PostHog makes your product [self-driving](/self-driving), and no, this isn't about cars. It digs through your product data, finds what's worth fixing, and has agents do the work. 
```

> It starts with context. Having a ton of developer tools in one place – [product analytics](/product-analytics), [web analytics](/web-analytics), [session replay](/session-replay), [feature flags](/feature-flags), [experiments](/experiments), [error tracking](/error-tracking), [logs](/logs), [AI observability](/ai-observability), and a [data warehouse](/data-stack) – means PostHog sees everything happening in your product. That's the part most other tools can't do: [loop engineering](/newsletter/loops) needs all kinds of signals from your product, and with PostHog, those are all in one place.

Reviewer: I forget where we have it, but I do like the

1. Developer tools
2. Data
3. Context warehouse (we're missing this here), but I realize having context twice in one sentence is a bit awkward.

Framing

I think we're overemphasizing loops with that last sentence.

```suggestion
It starts with context. Having a ton of developer tools in one place – [product analytics](/product-analytics), [web analytics](/web-analytics), [session replay](/session-replay), [feature flags](/feature-flags), [experiments](/experiments), [error tracking](/error-tracking), [logs](/logs), [AI observability](/ai-observability), and a [data warehouse](/data-stack) – means PostHog sees everything happening in your product.
```

> Then [scouts](/blog/what-is-a-scout) (small agents on a schedule) read across all of it and decide what's worth knowing and what’s just noise. What clears the bar becomes a report in your inbox, an agent picks it up, roots out the cause, and opens the PR. You review and merge.

Reviewer: I think the sentence better explains what scouts are than that. Also, there's more like "long-running agents on a schedule:
```suggestion
Then [scouts](/blog/what-is-a-scout) read across all of it and decide what's worth knowing and what’s just noise. What clears the bar becomes a report in your inbox, an agent picks it up, roots out the cause, and opens the PR. You review and merge.
```

> You can steer it from [Slack](/slack), the [web app](/ai), the [desktop app](/desktop), or your own editor via [the MCP](/mcp). Dealer's choice.

Reviewer: ```suggestion
You can steer it from [Slack](/slack), the [web app](/ai), the [desktop app](/desktop), or your own editor via [the MCP](/mcp) or [CLI](/docs/cli). Dealer's choice.
```

> Putting your product on self-driving mode means putting things like papercuts and conversion tweaks on cruise control, so you spend your time on the calls that need judgment – which is roughly [what software looks like by 2030](/newsletter/2030-shaped-software), when agents are the primary users and humans mostly approve, prioritize, and decide.

Author: ```suggestion
```

> Putting your product on self-driving mode means putting things like papercuts and conversion tweaks on cruise control, so you spend your time on the calls that need judgment – which is roughly [what software looks like by 2030](/newsletter/2030-shaped-software), when agents are the primary users and humans mostly approve, prioritize, and decide.

Reviewer: I also feel that the 2030 software call out is maybe a bit forced here. I think this is useful for us, but maybe not as useful as to be a selling point to others.
